alecs-theme/src/css/_imports.scss

```diff
@@ -0,0 +1,13 @@
+@import "node_modules/aos/src/sass/aos.scss";
```

Comment:
What is this ```aos.scss``` file?

Reply:
css that comes with the scroll animation library. [https://github.com/michalsnik/aos](url)